alfresco-transform-misc/alfresco-transform-misc-boot/src/test/java/org/alfresco/transformer/MiscControllerTest.java

```diff
@@ -408,7 +408,7 @@ public void textToPdf() throws Exception
         StringBuilder sb = new StringBuilder();
         for (int i = 1; i <= 5; i++)
         {
-            sb.append(i);
+            sb.append(Integer.toString(i));
             sb.append(" I must not talk in class or feed my homework to my cat.\n");

```

Comment:
Error found while working on TextToPdfContentTransformerTest which had the same problem.